py/client/pydeephaven/session.py

```diff
@@ -602,16 +616,10 @@ def fetch_table(self, ticket: SharedTicket) -> Table:
         Raises:
             DHError
         """
-        table = Table(session=self, ticket=ticket.api_ticket)
         try:
-            table_op = FetchTableOp()
-            return self.table_service.grpc_table_op(table, table_op)
+            return self.table_service.fetch_etcr(ticket.pb_ticket)
```

Comment:
I'm surprised we can't delegate to `fetch`.

Reply:
The generic `fetch` only return s an export ticket while `fetch_etcr` produces a response including some required table metadata needed for the `Table` constructor.